python/pyspark/sql/group.py

```diff
@@ -70,6 +70,9 @@ def __init__(self, jgd: JavaObject, df: DataFrame):
         self._df = df
         self.session: SparkSession = df.sparkSession
 
+    def __repr__(self) -> str:
+        return f"GroupedData{self._jgd.toString()[26:]}"
```

Comment:
I would put 26 to a separate variable, and add an assert if the truncated string starts with `[` ..

Reply:
If we'd be more conservative, we can invoke super if the string doesn't start with `[`.

Reply:
let me check again